examples/favorites_advanced/bloc_favorites_advanced/pubspec.yaml

```diff
@@ -37,6 +37,7 @@ dependencies:
   mockito: ^5.0.4
   declarative_animated_list: ^0.1.0-nullsafety.0
   clean_coverage: ^0.0.3
+  flow_builder: ^0.0.1
```

Comment:
@StanislavPrime, I believe there is a newer version

Reply:
I changed the version of  `flow_builder` to the newer 0.0.2